ihmc-perception/src/main/java/us/ihmc/perception/detections/yolo/YOLOv8DetectionResults.java

```diff
@@ -23,7 +23,7 @@ public class YOLOv8DetectionResults
    private final MatVector outputBlobs;
    private final RawImage detectionImage;
    private final FloatIndexer outputMasksIndexer;
-   private final Map<YOLOv8DetectionOutput, Mat> objectMasks = new HashMap<>();
+   private final Map<YOLOv8DetectionOutput, RawImage> objectMasks = new HashMap<>();
```

Comment:
Using RawImage instead of Mat just to be extra safe about memory allocation/deallocation here